src/components/Badge/BadgeReport.jsx

```diff
@@ -314,7 +314,7 @@ function BadgeReport(props) {
   const deleteBadge = () => {
     let newBadges = sortBadges.filter(badge => badge._id !== badgeToDelete._id);
     if (badgeToDelete.featured) {
-      setNumFeatured(--numFeatured);
+      setNumFeatured(prevNumFeatured => prevNumFeatured - 1);
     }
     setSortBadges(newBadges);
```

Comment:
similarly , we can reduce the redundancy here as well using : (if (badgeToDelete.featured) { setNumFeatured(prevNumFeatured => prevNumFeatured - 1); })

Reply:
Hi @navya9989. Thanks for your review! I feel you may be confusing the diff format, because this commit does make the code changes you suggested with no redundancy.